packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -250,6 +252,8 @@ curl -v -X GET \
 
 ##### Response example
 
+> **Note:** <ApiLifecycle access="ea" /> The `source` section and the `source` link in the response example below are only present in groups with type `APP_GROUP`. See [Group attributes](#group-attributes) and [Links object](#links-object).
```

Comment:
Nit: `groups of type APP_GROUP` (not "with type")

Reply:
updated here and everywhere else in this file